### ISSUE TITLE
iOS build failure: 'react_native_airship-Swift.h' file not found

### ISSUE THREAD

**Issue by Ackuq** [urbanairship/react-native-airship#495]
# Preliminary Info

### What Airship dependencies are you using?

```
"@ua/react-native-airship": "^15.2.1"`,
"airship-expo-plugin": "^1.0.0"
```

### What are the versions of any relevant development tools you are using?

```
"react": "18.2.0",
"react-native": "0.71.6"
```

Using Xcode 14.2 for building.

# Report

### What unexpected behavior are you seeing?

When building iOS compilation error occurs for the files `RTNAirshipMessageView.mm`, `RTNAirshipBootloader.m`, and, `RTNAirship.mm`:

```
› Compiling @ua/react-native-airship Pods/react-native-airship » RTNAirship.mm

❌  (node_modules/@ua/react-native-airship/ios/RTNAirship.mm:4:9)

  2 | 
  3 | #import "RTNAirship.h"
> 4 | #import "react_native_airship-Swift.h"
    |         ^ 'react_native_airship-Swift.h' file not found
  5 | 
  6 | @implementation RTNAirship
  7 | RCT_EXPORT_MODULE()
```

### What is the expected behavior?

Not erroring.

### What are the steps to reproduce the unexpected behavior?

Running a React Native project using `@ua/react-native-airship` and utilizing `airship-expo-plugin`. And having 
`useFrameworks` enabled in you `Podfile`.

### Workaround

I found a current workaround by patching the Objective C with the following change
```diff
- #import "react_native_airship-Swift.h"
+ #import <react_native_airship/react_native_airship-Swift.h>
```

### Potential solution
What I see some developers doing is to use the following if-statement when importing the Swift files into Objective C:

```
#if __has_include(<react_native_airship/react_native_airship-Swift.h>)
#import <react_native_airship/react_native_airship-Swift.h>
#else
#import "react_native_airship-Swift.h"
#endif
```

**Comment by rlepinski** [comment]
This looks great, could you please fill out https://github.com/urbanairship/react-native-airship/blob/main/.github/CONTRIBUTING.md and I will get it merged and released in the next version

**Comment by Ackuq** [comment]
> This looks great, could you please fill out https://github.com/urbanairship/react-native-airship/blob/main/.github/CONTRIBUTING.md and I will get it merged and released in the next version

Great! I've filled it out and submitted it 👍

**Comment by rlepinski** [comment]
Fixed in 15.2.2
